### PR TITLE
Bump fetch and add exports/globals for Headers, Request and Response

### PR DESCRIPTION
**Pull request by brentvatne**
Now on fetch 0.8.1, the latest tagged release. Previous version used was 0.7.0. See #1162 cc @vjeux @jtremback

### CODE REVIEW

**Comment by vjeux** [comment]
Need to update Libraries/JavaScriptAppEngine/Initialization/loadSourceMap.js as well

**Comment by jtremback** [comment]
:+1:

**Comment by vjeux** [comment]
@facebook-github-bot import

**Comment by vjeux** [comment]
Also need to update Examples/Movies/SearchScreen.js

**Comment by vjeux** [comment]
For those two, you probably want to use the global instead of requiring fetch directly

File: Libraries/JavaScriptAppEngine/Initialization/loadSourceMap.js [hunk 1]
```diff
@@ -17,8 +17,6 @@ var RCTSourceCode = require('NativeModules').SourceCode;
 var SourceMapConsumer = require('SourceMap').SourceMapConsumer;
 var SourceMapURL = require('./source-map-url');
 
-var fetch = require('fetch');
```
Comment (vjeux) [inline Libraries/JavaScriptAppEngine/Initialization/loadSourceMap.js]: can you make sure source maps are still working. trigger a redbox someway and make sure the file displayed is correct

Comment (brentvatne) [inline Libraries/JavaScriptAppEngine/Initialization/loadSourceMap.js]: :+1: this is working

**Comment by vjeux** [comment]
@facebook-github-bot import

**Pull request closed without merging**